### PR TITLE
template for route color contrast

### PR DESCRIPTION
**Pull request by aannie-y**
Add template for the routeColorContrastNotice.
Updated template with color box and string and collapsible button.
image

### CODE REVIEW

File: webapp/src/main/webapp/resources/template.soy [hunk 1]
```diff
@@ -552,3 +552,36 @@
...
+        <th>Route Text Color</th>
+      </tr>
+    </thead>
+    <tbody>
+      {foreach $notice in $notices}
+        <tr>
+          <td>{$notice.routeId}</td>
+          <td>{$notice.csvRowNumber}</td>
+          <td>{$notice.routeColor}</td>
+          <td>{$notice.routeTextColor}</td>
```
Comment (jupi3200) [inline webapp/src/main/webapp/resources/template.soy]: Can we make this more visual - for example adding a box with the correct color?

Comment (aannie-y) [reply webapp/src/main/webapp/resources/template.soy]: What do you mean the 'correct color'? As in do you want me to put the colour of the route text and the route?

Comment (jupi3200) [reply webapp/src/main/webapp/resources/template.soy]: As discussed, show a filled squest with the specified color.

Comment (aannie-y) [reply webapp/src/main/webapp/resources/template.soy]: Done! See updated image as per comment.

**Comment by aannie-y** [comment]
Updated look of the template (without collapsible CSS):
image

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**